satpy/dataset/__init__.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# Copyright (c) 2016-2019 Satpy developers
+#
+# This file is part of satpy.
+#
+# satpy is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later
+# version.
+#
+# satpy is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR
+# A PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# satpy.  If not, see <http://www.gnu.org/licenses/>.
+
+from .dataid import DataID, DataQuery, ModifierTuple, create_filtered_query  # noqa
+from .data_dict import get_key, DatasetDict  # noqa
+from .metadata import combine_metadata  # noqa
+from .anc_vars import dataset_walker, replace_anc  # noqa
```

Comment:
Are you sure about the relative imports? I prefer absolute imports myself.

Reply:
What do you mean "am I sure"? They work and for a subpackage I think it makes sense. I could see avoiding imports like `from ...readers.yaml_reader import X` or some other really long parent directory `.` sequence. Here I don't mind it. Is there a standard we can lean on?

Reply:
Ok